pypa/bandersnatch
# Add PEP691 Format support

- Move to 6.0.0.dev0 version
- Add config for setting formats to save to storage
  - `simple-format`
  - Valid options: ALL, HTML or JSON
- Move `mirror` simple generation functions to new simple module
- Add new tests to test_simple.py
  - Made tests be included with install by adding an __init__.py

Tests
- Add unittest coverage to SimpleAPI object
  - Test valid and invalid format requests
  - Test the JSON output for PEP691 compliance
    - Per pacakge
    - Global index
- Add check to Integration test to expect json index ...
